### ISSUE TITLE
problems about the Delta map and L map groundtruth

### ISSUE THREAD

**Issue by aqsc** [MengHao666/Minimal-Hand-pytorch#14]
delta_map = kin_chain[:, :, np.newaxis, np.newaxis].repeat(32, axis=-2).repeat(32, axis=-1)
It seems to make the value in W/H  the same. is it right? We only need to train the corresponding position of the groundtruth. But we find the position of maximum value of the predicted heatmap to match the groundthuth.

**Comment by MengHao666** [comment]
we find the position of maximum value in 2D heatmap as 2D keypoint position, but we find the corresponding location in delta map and location map according to location maximum value in 2D heatmap.

**Comment by aqsc** [comment]
When testing, we can do like this. But for training, don't we need to  training the corresponding location in delta map and L map according to location of maximum value of the groundtruth in 2D heatmap?

**Comment by MengHao666** [comment]
> When testing, we can do like this. But for training, don't we need to training the corresponding location in delta map and L map according to location of maximum value of the groundtruth in 2D heatmap?

In original paper, they make groundtruth deltamap and locationmap the same value.

**Comment by aqsc** [comment]
Ok, I havent seen this. Could you point out the sentense in the original paper? Thank you！

**Comment by MengHao666** [comment]
` tiling` means same value in my opinion.
![image](https://user-images.githubusercontent.com/47114466/121279882-e2aee980-c907-11eb-9bd8-3bfa9c5eb050.png)

**Comment by aqsc** [comment]
Ok， Thank you!  Another question, what do you think the difference between Delat map and PAfs. They seem the same.

**Comment by MengHao666** [comment]
I didn't know PAfs before , so I just have quick look about it. They seems similar as they both want to let network learn the structure relationship of the hand.
